.github/workflows/docker-unified.yml

```
@@ -24,7 +26,9 @@ env:
   #### IMPORTANT ####
 
   DOCKER_CACHE: "DEPOT"
-  DEPOT_PROJECT_ID: "${{ vars.DEPOT_PROJECT_ID }}"
+  DEPOT_PROJECT_ID: "s0gr1cr3jd"
```

@chakru-r can you review this merge datahub OSS PR; the project id should be got from github vars, right? anything else that should be restored?

```suggestion
  DEPOT_PROJECT_ID: "${{ vars.DEPOT_PROJECT_ID }}"
```

---

maybe we should also wrap with

```
  #### IMPORTANT ####
  #### THIS IS A CHANGE TO PREVENT OSS QUICKSTART INSTABILITY, DO NOT OVERWRITE THIS CHANGE IN MERGES ####
  #### IMPORTANT ####
  DEPOT_PROJECT_ID: "${{ vars.DEPOT_PROJECT_ID }}"
  #### IMPORTANT ####
  #### END CHANGES ####
  #### IMPORTANT ####
```
?

---

cool thanks!